tooling/camel-k-maven-plugin/src/test/java/org/apache/camel/k/tooling/maven/processors/GenerateYamlSupportClassesTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.k.tooling.maven.processors;
+
+import java.io.IOException;
+
+import com.squareup.javapoet.JavaFile;
+import org.apache.camel.k.tooling.maven.GenerateYamlSupportClasses;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class GenerateYamlSupportClassesTest {
+    @Disabled
+    @Test
+    public void testGenerateHasDataFormat() throws IOException {
+        JavaFile.builder("org.apache.camel.k.loader.yaml.parser", new GenerateYamlSupportClasses().generateHasDataFormat())
+            .indent("    ")
+            .build()
+            .writeTo(System.out);
```

Comment:
It could be better if we verify the out put content.

Reply:
yes, I still need to provide some more tests